doc/source/security/encrypting-system-tablespace.rst

```diff
@@ -5,8 +5,8 @@ Encrypting the System Tablespace
 ==================================================================
 
 |Percona Server| supports system tablespace encryption. The InnoDB system
-tablespace may be encrypted with the Master key encryption or may use keyring
-encryption with encryption threads. 
+tablespace may be encrypted with the master key encryption or the keyring
+encryption with advanced encryption key rotation.
```

Comment:
either we should mention keyring encryption is a tech preview feature. Or move this to a separate line under the tech preview heading.

Reply:
Added a line.